src/cmap/conn/stream_description.rs

```diff
@@ -35,6 +35,9 @@ pub(crate) struct StreamDescription {
 
     /// Whether the server associated with this connection supports the `hello` command.
     pub(crate) hello_ok: bool,
+
+    /// The maximum permitted size of a BSON wire protocol message.
+    pub max_message_size_bytes: Option<i32>,
```

Comment:
why `pub` and not `pub(crate)` like the other properties on this type? (...does that distinction even matter since the type itself is `pub(crate)`?)